dv-export/python/query-changes/main.py

```diff
@@ -0,0 +1,149 @@
+import argparse
+import json
+import logging
+from collections import OrderedDict
+from typing import Any
+
+from dotenv import dotenv_values
+from visier.connector import make_auth, VisierSession
+
+from constants import *
+from data_store import DataStore
+from dv_manager import DVManager
+from changes_fetcher import ChangesFetcher
+
+
+def setup_logger() -> logging.Logger:
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    formatter = logging.Formatter('%(asctime)s [%(levelname)s] %(name)s: %(message)s')
+
+    file_handler = logging.FileHandler('app.log', mode='a')
+    file_handler.setFormatter(formatter)
+    logger.addHandler(file_handler)
+
+    console_handler = logging.StreamHandler()
+    console_handler.setLevel(logging.INFO)
+    console_handler.setFormatter(formatter)
+    logger.addHandler(console_handler)
+
+    return logger
+
+
+logger = setup_logger()
+
+
+def parse_args() -> argparse.Namespace:
```

Comment:
Nit: Would these be reusable once you add 2c? If so, it might make sense to move them to a common directory

Reply:
I'll probably do this in 2c task. Since it will be cleaner what arguments will be added.